### PR TITLE
Add camel-ehcache integration test

### LINKED ISSUE #329: Fix and add camel-ehcache integration test
add EhCache integration test

### PR DESCRIPTION
**Pull request by f2par0**
fixes #329 
fix ehcache feature and add ehcache integration test

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -838,6 +838,7 @@
         <feature>scr</feature>
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <bundle dependency="true">mvn:org.ehcache/ehcache/${ehcache3-version}</bundle>
+        <bundle dependency="true">mvn:javax.cache/cache-api/1.1.0</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml:841]: Please use `${jcache-version}` instead, could you please also fix the hard-coded versions of cache-api elsewhere?

**Review by essobedo — COMMENTED** [review]
Just one small nitpick remark

File: features/src/main/feature/camel-features.xml (hunk 1 above)
Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: Ok done

**Review by essobedo — APPROVED** [review]
LGTM, thx

**Pull request merged**